src/biokbase/narrative/tests/data/app_specs.json

```diff
@@ -4113,5 +4113,279 @@
             "input": "null",
             "output": "no-display"
         }
+    },
+    {
```

Comment:
I replaced some existing example jobs with freshly-generated jobs from ee2 to ensure that the fields were up to date. These are the app specs for the apps used to generate those jobs.